dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/runtime/PagesDirectiveImportInfo.java

```diff
@@ -14,10 +14,21 @@
 
 import java.util.List;
 
-public interface JspImportInfo {
+/**
+ * Interface for retrieving page directive information as well as page import information.
+ */
+public interface PagesDirectiveImportInfo {
+
+    // Import Info methods
 
     List<String> getImportClassList();
 
     List<String> getImportPackageList();
 
+    List<String> getImportStaticList();
```

Comment:
I don't like having three methods for the one import, it would be nice to condense it into a single List (contains statics, package, and classes in separate groups).  Each directive would have a single method (but that's more future looking if there are new directives added) This ca be done later on.  Something to think about later on.